src/bika/lims/browser/worksheet/tools.py

```diff
@@ -38,7 +38,7 @@ def checkUserAccess(worksheet, request, redirect=True):
     """
     # Deny access to foreign analysts
     allowed = worksheet.checkUserAccess()
-    if allowed == False and redirect == True:
```

Comment:
I forgot this file in https://github.com/senaite/senaite.core/pull/2079

the change seemed small enough to be included here